src/shared/client/cg_api.cpp

```diff
@@ -472,6 +472,17 @@ bool trap_R_inPVS( const vec3_t p1, const vec3_t p2 )
 	return res;
 }
 
+std::vector<bool> trap_R_BatchInPVS(
+	const vec3_t origin,
+	const std::vector<std::array<float, 3>>& posEntities )
+{
+	std::array<float, 3> myOrigin;
+	VectorCopy(origin, myOrigin.data());
```

Comment:
Nit: `.data()` is not needed

Reply:
Right, this was mimicking what other functions do, I'll remove that and maybe we can do further PR to simplify code of other functions.